source/Cosmos.Common/StringHelper.cs

```diff
@@ -410,6 +410,16 @@ internal enum StringComparisonResultEnum {
             }
             return (int)StringComparisonResultEnum.Equal;
         }
+
+        /// <summary>
+        /// Check if string is numeric.
+        /// </summary>
+        /// <param name="aString">A string to check if numeric.</param>
+        /// <returns>Returns TRUE if string is numeric.</returns>
+        public static bool IsNumeric(string aString) {
+            int i = 0;
+            return int.TryParse(aString, out i);
```

Comment:
While this test works for short strings, it will not work for arbitrarily long strings.

Reply:
Fixed it, changed the method